scripts/qa_crowbarsetup.sh

```diff
@@ -1609,20 +1609,61 @@ function set_proposalvars()
     # Determine if we went through an upgrade
     [ -f /etc/cloudsource ] && export cloudsource=$(</etc/cloudsource)
 
-    wantswift=1
-    [ -z "$want_swift" ] && wantceph=1
-    [[ -n "$wanthyperv" ]] && wantswift= && wantceph= && networkingmode=vlan
+    ### dynamic defaults
+    case "$nodenumber" in
+        0|1)
+            deployswift=
+            deployceph=
+        ;;
+        2)
+            deployswift=1
+            deployceph=
+        ;;
+        *)
+            deployswift=
+            deployceph=1
+        ;;
+    esac
+
+    ### filter (temporarily changing defaults)
+    # F1: hyperV only without swift and ceph
+    [ -n "$wanthyperv" ] && deployswift= && deployceph= && networkingmode=vlan
```

Comment:
I agree
( I should have had another look at the branch name and done an overall cleanup :) )